UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/SettingsOverrides.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using MLAgents;
+
+public class SettingsOverrides : MonoBehaviour
+{
+    // Original values
+    Vector3 m_OriginalGravity;
+    float m_OriginalFixedDeltaTime;
+    float m_OriginalMaximumDeltaTime;
+    int m_OriginalSolverIterations;
+    int m_OriginalSolverVelocityIterations;
+
+    public float gravityMultiplier = 1.0f;
```

Comment:
I like.
I would add some helper [tooltips](https://docs.unity3d.com/ScriptReference/TooltipAttribute.html) or comments.

Reply:
Copied the descriptions of the timestep and solver settings from the docs.